wallet/wallet.go

```diff
@@ -29,21 +29,16 @@ type Wallet struct {
 	Net           libutils.NetworkType
 }
 
-// NewWallet initializies an new Wallet instance.
+// NewWallet initializes an new Wallet instance.
 // The Wallet is not loaded until LoadWallets is called.
-func NewWallet(root, net, version, logFolder string, buildDate time.Time) (*Wallet, error) {
+func NewWallet(root string, net libutils.NetworkType, version, logFolder string, buildDate time.Time) (*Wallet, error) {
 	if root == "" {
 		return nil, fmt.Errorf("root directory cannot be empty")
 	}
 
-	resolvedNetType := libutils.ToNetworkType(net)
-	if resolvedNetType == libutils.Unknown {
-		return nil, fmt.Errorf("network type is not supportted: %s", net)
-	}
```

Comment:
@itswisdomagain re https://github.com/crypto-power/cryptopower/pull/238#discussion_r1382516892, The "testnet3" string was corrected here(libutils.ToNetworkType(net) will return "testnet" for "testnet3") so I'm sure we are on the right part.